internal/components/components.go

```diff
@@ -228,6 +229,7 @@ func Components() (otelcol.Factories, error) {
 		bigipreceiver.NewFactory(),
 		carbonreceiver.NewFactory(),
 		chronyreceiver.NewFactory(),
+		cloudflarereceiver.NewFactory(),
```

Comment:
you should check out how to add your receiver now to this list, this has changed.

Reply:
updated